src/pages/GetInvolved/GetInvolved.js

```diff
@@ -1,10 +1,46 @@
-import React from 'react';
-import Layout from 'components/Layout/Layout';
+import React from "react";
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import Layout from "components/Layout/Layout";
+
+const createMarkup = (body) => {
+  return {__html: body};
+}
+
+const GetInvolved = ({data}) => {
+  let clubDataAll = useSelector((state) => state.clubs);
+
+  if (!data) {
+    return null;
+  } else {
+    data = data.join_a_club_page;
```

Comment:
woah woah woah - you are mutating state here, in js this is a no no for this kind of situation where children are dependent on state.